src/api/layers/ExternalGroupOfLayers.class.js

```diff
@@ -58,7 +58,9 @@ export default class ExternalGroupOfLayers extends ExternalLayer {
 
     getID() {
         // format coming from https://github.com/geoadmin/web-mapviewer/blob/develop/adr/2021_03_16_url_param_structure.md
-        return `GRP|${this.baseURL}|${this.externalLayerId}`
+        // NOTE we don't differentiate between group of layers and regular WMS layer. This differentiation was not
+        // done the legacy parameter and is not required.
+        return `WMS|${this.baseURL}|${this.externalLayerId}`
```

Comment:
Why reverting the distinction between WSM and GRP of layers in the process?
Couldn't we, technically, also have a group of WMTS layer?

Reply:
On the old viewer we did not have any distinction, see comment above. Having the distinction would need a dirty hack to handle legacy group of layers, see commit https://github.com/geoadmin/web-mapviewer/pull/584/commits/1db3dce307efaa50ce8c442175d1dd2b35ed9029

Reply:
Couldn't we let it as `isLoading` at startup, load the getCapabilites (we've got one even within the legacy param, right?) and check what it really is?

I'd really like to keep this concept of groups of layers, as we might move to other type of external layers (WMTS, vector?)

Reply:
But that would still require the dirty hack for legacy url that cannot be handle in the legacy params plugin then. I don't think we need the distinction in the url if this is a group of layer or not, unless we support in future mix of group of layers, but I don't this that would work, having a url that contains both WMTS and WMS.

Reply:
If you look at the code not having this distinction in the URL make everything simpler, less hacky and error prone in my opinion.